stacks-blockchain/Stacks-testnet.toml

```diff
@@ -1,7 +1,7 @@
 [node]
 rpc_bind = "0.0.0.0:20443"
 p2p_bind = "0.0.0.0:20444"
-bootstrap_node = "047435c194e9b01b3d7f7a2802d6684a3af68d05bbf4ec8f17021980d777691f1d51651f7f1d566532c804da506c117bbf79ad62eea81213ba58f8808b4d9504ad@testnet.stacks.co:20444"
+bootstrap_node = "029266faff4c8e0ca4f934f34996a96af481df94a89b0c9bd515f3536a95682ddc@seed.testnet.hiro.so:20444"
 # working_dir = "/tmp/stacks-blockchain-data"
 
 
```

Comment:
@CharlieC3 what about the reference below to `bitcoind.testnet.stacks.co`, should these be updated to a `hiro.so` domain?